portfolio/src/main/webapp/script.js

```diff
@@ -57,3 +57,37 @@ function getText() {
 }
 
 
+//wk3pt6
+
+
+function getComments() {
+  
+  document.getElementById("history").innerText = '';
+  
+  const maxval = document.getElementById('max').value;
```

Comment:
This function might be more reusable if you made maxval an argument.  For example, what if you had multiple inputs that all set the max number of comments?  This is somewhat common for these kinds of UIs, where you could set limits at both the top and the bottom of the comments section

Reply:
so I should take this line out of the function and make maxval a parameter? If I did this how would the button in the html know where to get the max from?

Reply:
"this" in an onchange string refers to the event.  so something like "onchange=fun(this.value)" would pass the new value to "fun".  That's only if you take my suggestion above though, otherwise this is fine